utils/update_doc_version.sh

```diff
@@ -4,7 +4,7 @@
 # Example: Suppose that before running this script, the build.gradle reads 7.8.9.  This script will replace 
 # all occurrences of #.#.# with 7.8.9 in the README files.
 
-# Get the current maven project version.
+# Get the current project version.
 RELEASED_VERSION=$(./gradlew properties | grep -o 'version.*' | sed -n -e 's/^.*version: //p')
```

Comment:
it could be made into a gradle task, it doesn't need to be a script - it could be called like this:

`./gradlew version`